06.bowling_object/lib/frame.rb

```diff
@@ -0,0 +1,56 @@
+# frozen_string_literal: true
+
+require_relative 'shot'
+
+class Frame
+  attr_accessor :shots
+  attr_reader :number
+
+  def initialize(number)
+    @shots = []
+    @number = number
+  end
+
+  def add(pin)
+    @shots = [*@shots, Shot.new(pin)]
+  end
+
+  def score(frames)
+    score = pins.sum
+    if @number < 9
+      score += next_two_shots(frames, @number) if strike?
+      score += next_one_shots(frames, @number) if spare?
+    end
+    score
+  end
+
+  def pins
+    @shots.map(&:score)
+  end
+
+  def two_shots_done?
```

Comment:
```suggestion
  def addable?
```

追加可能か？みたいな名前の方がいいかも。（ストライクの時は1回しか投げないので）

Reply:
たしかに2投するとは限りませんね💦 修正しました